src/routes/+page.svelte

```diff
@@ -300,7 +300,7 @@
 			<div class="flex items-end justify-end">
 				<Button type="minimal" class="gap-2 text-end text-lg hover-child:translate-x-1">
 					{solutions.slice(-1)[0]?.description ?? ""}
-					<ChevronRight class="h-4 w-4 transition-transform duration-500" />
+					<ChevronRight class="w-10 h-10 transition-transform duration-500" />
```

Comment:
Also are you sure it passes the `prettier-plugin-tailwindcss` linting? You inverted `h-*` and `w-*`, that's weird the CI didn't say anything...

Reply:
Update: don't worry! It's normal that Prettier didn't say anything, the [TailwindCSS plugin is broken when used with other plugins on Prettier 3](https://github.com/tailwindlabs/prettier-plugin-tailwindcss#compatibility-with-other-prettier-plugins) (first line + issue 186)! So it will be likely fixed by another PR later but for now we can't do anything about it :/